README.md

```diff
@@ -14,6 +14,26 @@ Argo Helm is a collection of **community maintained** charts for [https://argopr
 helm repo add argo https://argoproj.github.io/argo-helm
 ```
 
+## Version Support Policy
+As our project is maintained by a small team, we must focus our limited resources on following upstream projects and ensuring the stability of the latest version.
+
+Consequently, **we do not provide bug fixes or security patches for older versions.** Our official support is limited to **the latest version of the upstream projects** only.
+
+We strongly encourage all users to upgrade to the latest version to benefit from the most recent features, bug fixes, and security patches.
+
+### For Users Unable to Upgrade
+If you are unable to upgrade to the latest version due to specific constraints, you can manually override the image tag to use a specific version.
+
+Example for [argo-cd](https://github.com/argoproj/argo-helm/blob/15658c3c0aeaa2f8fc5ef77555c78e5ce2256e12/charts/argo-cd/values.yaml#L62):
+```yaml
+global:
+  image:
+    tag: v3.1.3
+```
+
+### How You Can Help
+This policy may evolve as our team grows. If you are interested in becoming a maintainer and helping us expand our support capabilities, we encourage you to read our [contribution guidelines](CONTRIBUTING.md).
```

Comment:
given that link is a line below, and you mention becoming a maintainer, I'd link here: https://github.com/argoproj/argoproj/blob/main/community/membership.md

... which helpfully doesn't include our project.

Reply:
Wow I didn't know it... Thank you! will fix it.